docs/kyma/04-11-local-reinstallation.md

```diff
@@ -7,12 +7,12 @@ The custom scripts allow you to remove Kyma from a Minikube cluster and reinstal
 
```

Comment:
Line 6 talks about scripts. I don't see any scripts used in this doc, only CLI commands reference.